manifests/init.pp

```diff
@@ -35,6 +35,14 @@
 #   * sysstat
 #   * udev-post
 #
+# You may override the default list in hiera, with the use of a
+# knockout prefix '--'. For example, to remove sshd from the default
+# whitelist:
+#
+# ---
+# svckill::ignore_defaults:
```

Comment:
This shoould just be ignore, not ignore_defaults. it will work the same way since they are being appended to one another.